Fixes undefined symbol _fp_hw required by the ABI:
readelf -s /opt/crosstool/i686-pc-linux-gnu/i686-unknown-linux-gnu/gcc-3.4.3-glibc-2.2.2/i686-unknown-linux-gnu/lib/crt1.o | grep fp_hw
    29: 00000000     4 NOTYPE  GLOBAL DEFAULT  UND _fp_hw 
Rumored to fix problems with icc, which gets upset at any symbol
which is undefined, even if nobody cares about it.

Revision 1.16, Thu Feb 22 19:57:34 2001 UTC (4 years, 2 months ago) by drepper
Branch: MAIN
CVS Tags: glibc-2_2_3
Changes since 1.15: +1 -1 lines
Diff to previous 1.15 (colored)

(_fp_hw): Actually define label.

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/i386/elf/start.S,v
retrieving revision 1.15
retrieving revision 1.16
diff -u -r1.15 -r1.16
--- libc/sysdeps/i386/elf/start.S	2001/02/03 21:34:54	1.15
+++ libc/sysdeps/i386/elf/start.S	2001/02/22 19:57:34	1.16
@@ -85,7 +85,7 @@
    meaningless since we don't support machines < 80386.  */
 	.section .rodata
 	.globl _fp_hw
-	.long 3
+_fp_hw:	.long 3
 	.size _fp_hw, 4
 
 /* Define a symbol for the first piece of initialized data.  */
